python/lsst/analysis/tools/analysisPlots/skyObject.py

```diff
@@ -61,16 +61,27 @@ def setDefaults(self):
         self.prep.selectors.skyObjectSelector = SkyObjectSelector()
 
         self.process.buildActions.hist_psf_flux = LoadVector(vectorKey="{band}_psfFlux")
-        self.process.buildActions.hist_09_flux = LoadVector(vectorKey="{band}_ap09Flux")
+        self.process.buildActions.hist_ap09_flux = LoadVector(vectorKey="{band}_ap09Flux")
+        self.process.buildActions.hist_gaap1p0_flux = LoadVector(vectorKey="{band}_gaap1p0Flux")
         self.process.buildActions.hist_psf_sn = SNCalculator(fluxType="{band}_psfFlux")
-        self.process.buildActions.hist_09_sn = SNCalculator(fluxType="{band}_ap09Flux")
+        self.process.buildActions.hist_ap09_sn = SNCalculator(fluxType="{band}_ap09Flux")
+        self.process.buildActions.hist_gaap1p0_sn = SNCalculator(fluxType="{band}_gaap1p0Flux")
 
         self.produce = HistPlot()
 
         self.produce.panels["panel_flux"] = HistPanel()
         self.produce.panels["panel_flux"].label = "Flux (nJy)"
-        self.produce.panels["panel_flux"].hists = dict(hist_psf_flux="psfFlux", hist_09_flux="ap09Flux")
+        self.produce.panels["panel_flux"].hists = dict(
+            hist_psf_flux="psfFlux", hist_ap09_flux="ap09Flux", hist_gaap1p0_flux="gaap1p0Flux"
+        )
```

Comment:
As above, for me, `black` recommends this type of format, which I do think is cleaner:
```
        self.produce.panels["panel_flux"].hists = dict(
            hist_psf_flux="psfFlux",
            hist_ap09_flux="ap09Flux",
            hist_gaap1p0_flux="gaap1p0Flux",
        )
```